docs/integrations/sources/slack.md

```diff
@@ -49,7 +49,44 @@ The Slack connector should not run into Slack API limitations under normal usage
 
 ### Setup guide
 
-Generate a API access token using the [Slack documentation](https://slack.com/intl/en-ua/help/articles/215770388-Create-and-regenerate-API-tokens)
+{% hint style="info" %}
+If you are using an "legacy" Slack API, skip to the Legacy API Key section below.
+{% endhint %}
+
+In order to pull data out of your Slack instance, you need to create a Slack App. This may sound daunting, but it is actually pretty straight forward. Slack supplies [documentation](https://api.slack.com/start) on how to build apps. Feel free to follow that if you want to do something fancy. We'll describe the steps we followed to creat the Slack App for this tutorial.
+
+{% hint style="info" %}
+This tutorial assumes that you are an administrator on your slack instance. If you are not, you will need to coordinate with your administrator on the steps that require setting permissions for your app.
+{% endhint %}
+
+1. Go to the [apps page](https://api.slack.com/apps)
+1. Click "Create New App"
+1. It will request a name and the slack instance you want to create the app for. Make sure you select the instance form which you want to pull data.
+1. Completing that form will take you to the "Basic Information" page for your app.
+1. Now we need to grant the correct permissions to the app. (This is the part that requires you to be an administrator). Go to "Permissions". Then under "Bot Token Scopes" click on "Add an OAuth Scope". We will now need to add the following scopes:
+    ```
+    channels:history
+    channels:join
+    channels:read
+    files:read
+    groups:read
+    links:read
+    reactions:read
+    remote_files:read
+    team:read
+    usergroups:read
+    users.profile:read
+    users:read
+    ```
+    This may look daunting, but the search functionality in the dropdown should make this part go pretty quick.
+1. Scroll to the top of the page and click "Install to Workspace". This will generate a "Bot User OAuth Access Token". We will need this in a moment.
+1. Now go to your slack instance. For any public channel go to info => more => add apps. In the search bar search for the name of your app. (If using the desktop version of slack, you may need to restart Slack for it to pick up the new app).
```

Comment:
Does it mean you can only pull data from the channels where the bot is installed?

Reply:
yes. unfortunately that is a current limitation of our slack source.